schemas/representations.md

```diff
@@ -291,3 +306,44 @@ This is because inline unions are only a defined concept when working with types
 that have a map representation -- so, our `Bar` type in the previously examples,
 which was of `int` kind, doesn't work for this example.  We replaced it with
 another struct type, which -- since it has a `map` representation -- works.
+
+### enum string representation
+
+By default, an enum is simply represented as a string in the data model. An
+`enum` in the schema simply defines the list of possible strings that could be
+used in that position.
+
+#### Example
+
+```ipldsch
+type Status enum {
+	| "NOPE"
+	| "YEP"
+	| "MAYBE"
```

Comment:
Hm.  Should we maybe ... infer strings as the serial form by default, but make the syntax for all the representations use something that looks more "like" a TypeName?

E.g.:

```
type Status enum {
	| Nope
	| Yep
	| Maybe
}
```

and this is equivalent to:

```
type Status enum {
	| Nope "Nope"
	| Yep "Yep"
	| Maybe "Maybe"
}
```

I'm thinking of this in part because of the need of codegen to generate reasonable symbols, such as in this case:

```go
const (
	Status_Nope  = Status("Nope")
	Status_Yep   = Status("Yep")
	Status_Maybe = Status("Maybe")
)
```

Reply:
Yes, I was heading in that direction when I introduced `type EnumValue string` to mimmick `type TypeName string` which is unquoted, but backed out some of my changes. It's awkward not having ways to express this in schema-schema.
If you're OK with this then I can go ahead and unquote.